kpet/data.py

```diff
@@ -563,6 +565,14 @@ def get_sets_from_regex_list(regex_list):
                                   "in suite: {}\ncase: {}".
                                   format(suite.name, case.name))
 
+                # Make sure there's at least a maintainer (suite- or
+                # case-level)
+                if not case.maintainers and not suite.maintainers:
+                    raise Invalid("There has to be a maintainer for each "
+                                  "test, either at suite level or at case "
+                                  "level; in suite: {}\ncase: {}".
+                                  format(suite.name, case.name))
+
```

Comment:
This is an inappropriate place for such a check, if only because it has nothing to do with sets. Instead please put this into `kpet.data.Suite.__init__()`